oecophylla/taxonomy/taxonomy.rule

```diff
@@ -258,7 +258,7 @@ rule taxonomy_centrifuge:
                     -p {threads} \
                     -S {temp_dir}/map.tmp \
                     --report-file {output.profile} \
-                    2>> {log} 1>&2
+                    2> {log} 1>&2
```

Comment:
What's the purpose of this edit? If there are multiple commands in one rule, `2> {log} 1>&2` will overwrite all previous logs.

Reply:
The first write in a rule should be single carrot. Otherwise if you run rule twice it gets appended.

Reply:
Oh yes you are correct. Sorry. I think I can merge it now.